packages/icon/src/createGlyphComponent.tsx

```diff
@@ -47,20 +68,21 @@ export default function createGlyphComponent(
           className,
         )}
         title={getGlyphTitle(glyphName, title)}
-        height={size}
-        width={size}
+        height={typeof size === 'number' ? size : sizeMap[size]}
```

Comment:
Can we move this above and set this to a variable to avoid repeating this logic?

Reply:
👍